integration-tests/tests/src/tests/sync/user.ts

```diff
@@ -67,7 +67,11 @@ function removeExistingUsers(): void {
     if (this.app) {
       const users = this.app.allUsers;
       for (const userId of Object.keys(this.app.allUsers)) {
-        await this.app.removeUser(users[userId]);
+        if (users[userId].isLoggedIn) {
+          await this.app.deleteUser(users[userId]);
+        } else {
+          await this.app.removeUser(users[userId]);
+        }
```

Comment:
Why was this change needed? From the TS docs, `removeUser()` will handle logging out the user.

Reply:
This was an attempt to fix the test when I was unsure why a user object was being returned for something that should have failed.  The first assumption was that the user actually did exist.  This can be removed

Reply:
Ah okay, thanks for explaining!